client/blocks/login/index.jsx

```diff
@@ -284,10 +284,10 @@ class Login extends Component {
 			}
 
 			if ( isJetpackCloudOAuth2Client( oauth2Client ) ) {
-				headerText = translate( 'Log in to Jetpack.com with your WordPress.com account.' );
+				headerText = translate( 'Howdy! Log in to Jetpack.com with your WordPress.com account.' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.